qa-pipelines/tasks/lib/cf-deploy-upgrade-common.sh

```diff
@@ -26,7 +26,19 @@ fi
 # Check that the kube of the cluster is reasonable
 bash ${CAP_DIRECTORY}/kube-ready-state-check.sh kube
 
-PROVISIONER=$(kubectl get storageclasses persistent -o "jsonpath={.provisioner}")
+if kubectl get sc | grep "persistent" > /dev/null ; then
+    STORAGECLASS="persistent"
+    PROVISIONER=$(kubectl get storageclasses ${STORAGECLASS} -o "jsonpath={.provisioner}")
+    cap_platform=$(kubectl get configmap -n kube-system cap-values -o json | jq -r .data.platform)
+    garden_rootfs_driver=$(kubectl get configmap -n kube-system cap-values -o json | jq -r '.data["garden-rootfs-driver"] // "btrfs"')
+elif [[ ${cap_platform} == "eks" ]] && kubectl get sc | grep gp2 > /dev/null ; then
+    STORAGECLASS="gp2"
```

Comment:
use the `gp2` storageclass definition to create `persistent`

Reply:
My above comment explains, why not to do this.